src/components/MessageInput/__tests__/MessageInput.test.js

```diff
@@ -0,0 +1,479 @@
+import React from 'react';
+import { cleanup, render, waitFor, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import axios from 'axios';
+import MessageInput from '../MessageInput';
+import { Chat } from '../../Chat';
+import { Channel } from '../../Channel';
+import {
+  generateChannel,
+  generateMember,
+  generateUser,
+  generateMessage,
+  useMockedApis,
+  getOrCreateChannelApi,
+  getTestClientWithUser,
+} from '../../../mock-builders';
+
+jest.mock('axios');
+let chatClient, channel;
+
+// mock i18n
+const t = (t) => t;
+
+// silence console.warn because some components log large warnings in catch statements
+console.warn = () => {};
```

Comment:
I think it is better to do it inside the test statements that output those large warnings. E.g: `jest.spyOn(console, 'warn').mockImplementationOnce(() => null);` This way we prevent ignoring warns that would be relevant.

Reply:
Totally agree, will fix